ragaai_catalyst/tracers/agentic_tracing/upload/session_manager.py

```
@@ -0,0 +1,156 @@
+import logging
+import os
+import threading
+
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+from urllib3.exceptions import PoolError, MaxRetryError, NewConnectionError
+from requests.exceptions import ConnectionError, Timeout
+from http.client import RemoteDisconnected
+from ragaai_catalyst import RagaAICatalyst
```

Flake8 (7.2.0)

[error] 'ragaai_catalyst.RagaAICatalyst' imported but unused

(F401)